src/plugins/intel_gna/common/versioning.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "versioning.hpp"
+
+#include <sstream>
+#include <string>
+
+#include "openvino/core/version.hpp"
+
+namespace GNAPluginNS {
+namespace common {
+
+std::string GetVersionOfOv() {
```

Comment:
```suggestion
std::string get_openvino_version_string() {
```

Reply:
Is there some specific requirement for function naming,
I thought we use "Google C++ Style Guide"?

According to https://google.github.io/styleguide/cppguide.html#Function_Names
"Regular functions have **mixed case**; accessors and mutators may be named like variables."

Reply:
@nkogteva please clarify regarding the style

Reply:
@kbruniec As far I as know, openvino new code uses snake_case, but we still have a lot of code with CamelCase. I'm trying to use new code style in new code. But if you have some doubts, you can keep it as is? we already have a mess. Code style aligning can be done in separate PR later.